data/src/main/java/me/androidbox/data/worker_manager/FullAgendaSynchronizerImp.kt

```diff
@@ -0,0 +1,45 @@
+package me.androidbox.data.worker_manager
+
+import androidx.work.BackoffPolicy
+import androidx.work.Constraints
+import androidx.work.ExistingPeriodicWorkPolicy
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequestBuilder
+import androidx.work.WorkManager
+import androidx.work.WorkRequest
+import me.androidbox.domain.work_manager.FullAgendaSynchronizer
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+class FullAgendaSynchronizerImp @Inject constructor(
+    private val workManager: WorkManager) : FullAgendaSynchronizer {
+
+    companion object {
+        const val THIRTY_MINUTES = 30L
```

Comment:
I Would rename it to "SYNC_INTERVAL" or similar
What if in the future you change it to 15 min or 1 hour? You need to also rename a variable

Reply:
@MKiperszmid updated